examples/hacker-news-analyzer/package.json

```diff
@@ -7,22 +7,23 @@
     "node": ">=18.0.0"
   },
   "scripts": {
-    "dev": "tsx ./src/index.tsx",
-    "start": "npx gensx start ./src/workflows.tsx",
-    "build": "tsc",
+    "dev": "nodemon",
+    "start": "pnpm build && node dist/index.js",
+    "build": "tsc -p tsconfig.build.json",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "deploy": "npx gensx deploy -ev OPENAI_API_KEY ./src/workflows.tsx"
+    "prepare": "ts-patch install -s"
```

Comment:
Probably worth discussing what the ideal set of commands to have here is. I had gone through and updated this example so that `start` ran the dev server and `deploy` would do the deployment

Reply:
is running `pnpm prepare` a prereq now?

Reply:
Yes for projects that use decorators.

Reply:
We can definitely replace nodemon with the dev server.

Reply:
I had done it the other way becasue it felt weird for `pnpm dev` to run `gensx start`. Could always change the CLI command name though

Reply:
We could make it `gensx serve` or something.